pdns/sdig.cc

```diff
@@ -259,14 +278,35 @@ try {
         xpfsrc = argv[++i];
         xpfdst = argv[++i];
       }
-      if (strcmp(argv[i], "class") == 0) {
+      else if (strcmp(argv[i], "class") == 0) {
         if (argc < i+2) {
           cerr << "class needs an argument"<<endl;
```

Comment:
I know this isn't you, but mostly they do this
```suggestion
          cerr << "class needs an argument" << endl;
```